foodsaving/groups/models.py

```diff
@@ -44,3 +44,6 @@ def add_member(self, user, history_payload=None):
     def remove_member(self, user):
         pre_group_leave.send(sender=self.__class__, group=self, user=user)
         self.members.remove(user)
+
+    def is_member(self, user):
+        return user in self.members.all()
```

Comment:
`self.members.filter(user=user).exists()`? Otherwise it's instantiating model objects for all the users (unless python/django allows something fancy)

Reply:
.. oh, forgot to mark it as review comment. This one is now.